packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.6.0
+
+As part of implementing federated architecture and making the interface compatible for other platform this version contains the following **breaking changes**:
+
+* Changes to the platform agnostic interface:
+  * The `InAppPurchaseConnection` class has been renamed to `InAppPurchase`. To access the generic InApp Purchase API use `InAppPurchase.instance`;
```

Comment:
nits: maybe we can make this clearer such as


If you used `inAppPurchaseConnection.instance` to access generic In App Purchase APIs, please use `InAppPurchase.instance` instead.

Reply:
Done